### DISCUSSION TITLE
DAI 2024-2025 - My picocli project - Mário André ROCHA FERREIRA

### DISCUSSION THREAD

**Discussion by Thynkon** [heig-vd-dai-course/heig-vd-dai-course#293] [Show and tell]
https://github.com/Thynkon/dai-lab01-test

**Comment by ludelafo** [comment]
The directory `.settings` seems a bit weird to me. Do you know how it got there?  The files `.classpath` and `.project` seem weird as well.

The rest seems OK to me.

**Reply by Thynkon** [reply]
I use neovim as my IDE and I think those files where created by the LSP I use. For the next practical works, I will add both files to ``.gitignore``.
